doc/getting_started/index.rst

```diff
@@ -382,5 +420,19 @@ Build the hypervisor and device model
 
    The build results are found in the ``build`` directory.
 
+#. Build the ACRN tools (included in the acrn-hypervisor repo):
+
+   .. code-block:: console
+
+      $ cd ../tools
```

Comment:
How about this instead (and it will work if new tools show up later):

    $ cd ../tools
    $ for d in */; do make -C "$d"; done

Reply:
I was thinking about this but was concerned it was too obscure for people not familiar with scripts... but I agree it's more robust and simplifies the instruction set. I have changed this.